peachjam/views/courts.py

```diff
@@ -65,25 +73,23 @@ def add_facets(self, context):
             context["facet_data"]["judges"] = {
                 "label": Judge.model_label_plural,
                 "type": "checkbox",
-                "options": judges,
+                "options": [(j, j) for j in judges],
                 "values": self.request.GET.getlist("judges"),
             }
 
         if "outcomes" not in self.exclude_facets:
-            outcomes = list(
-                outcome
-                for outcome in self.form.filter_queryset(
+            outcomes = Outcome.objects.filter(
+                pk__in=self.form.filter_queryset(
                     self.get_base_queryset(), exclude="outcomes"
                 )
                 .order_by()
-                .values_list("outcomes__name", flat=True)
+                .values_list("outcomes__id", flat=True)
                 .distinct()
-                if outcome
             )
             context["facet_data"]["outcomes"] = {
                 "label": _("Outcomes"),
                 "type": "checkbox",
-                "options": outcomes,
+                "options": [(outcome.code, outcome.name) for outcome in outcomes],
```

Comment:
i've reverted this to outcome.name. We should add the code for outcome and other facets, like matter type, so that we can facet across multiple languages. I'm not going to include that change in this PR.